src/components/org/apache/jmeter/visualizers/backend/influxdb/InfluxdbBackendListenerClient.java

```diff
@@ -318,6 +335,13 @@ public void setupTest(BackendListenerContext context) throws Exception {
                 }
             }
         }
+        // Check if more fields are filled ( corresponding to user tag )
+        context.getParameterNamesIterator().forEachRemaining(name->{
+            if ( !defaultArg.containsKey(name)) {
+                TAG_USER += "," + AbstractInfluxdbMetricsSender.tagToStringValue(name.trim()) + "=" + AbstractInfluxdbMetricsSender.tagToStringValue(context.getParameter(name).trim());
```

Comment:
Add some logging here to debug in case of issue

Reply:
Done